Makefile

```
@@ -117,8 +117,9 @@ test-short:
 
 ## test-e2e: Run end to end tests via knuu.
 test-e2e:
-	@echo "--> Running e2e tests on version: $(shell git rev-parse --short HEAD)"
-	@KNUU_NAMESPACE=test E2E_VERSION=$(shell git rev-parse --short HEAD) E2E=true go test ./test/e2e/... -timeout 10m -v
+	@export E2E_VERSION=$(git tag -l)
```

[quesiton] `git tag -l` appears to list all versions. Does this env variable contain all possible versions and not just a particular version?

```suggestion
	@export E2E_VERSIONS=$(git tag --list)
```

---

[nit] Makes sense. This is super nit picky but then I think plural `E2E_VERSIONS` is more appropriate

---

Yeah I can change that